wiki/Help_centre/Account_restrictions/en.md

```diff
@@ -68,7 +68,7 @@ If we decide to let you return, we will require that you agree to some terms bef
 - **You should not associate with other accounts** including playing on shared computers.
 - **Any major misbehaviour will result in immediate restriction**. This includes major silences.
 
-In some cases, an appeal returnee may be banned from official and badged tournaments, both as a competitor and member of the tournament's team. The duration of this ban is 1 year per offence (if a user is on their second appeal, the ban lasts 2 years). The unrestriction email will clarify the duration of this ban.
+In most cases, an appeal returnee will be banned from official and badged tournaments, both as a competitor and member of the tournament's team. The duration of this ban is 1 year per offence (if a user is on their second appeal, the ban lasts 2 years). The unrestriction email will clarify the duration of this ban.
```

Comment:
> The duration of this ban is 1 year per offence (if a user is on their second appeal, the ban lasts 2 years).

this information is not present in the [tournament bans article](https://osu.ppy.sh/wiki/en/Help_centre/Tournament_bans) 🤔

Reply:
Tournament bans themself are not appealable, only the restriction/silence -> restriction. So there would be no "second" appeal for strictly a tournament ban, or even a first one. Silences can lead to restrictions if heinous enough, iirc.

![NzGJaHM](https://user-images.githubusercontent.com/90729492/213894934-cf1ec7a9-798a-40db-a883-f8c237019a29.png)


Don't really see a reason for it to be included. Tournament bans can be added to someone regardless if they've been restricted/silenced.

Reply:
The line here is saying that for a second appeal (i.e. offence, from a second regular restriction) then the tournament ban from returning from that second restriction would be 2 years, if i'm understanding this correctly.

This information is not present in the tournament bans article.

> Tournament bans can be added to someone regardless if they've been restricted/silenced.

Right but this line is about automatic tournament bans when someone gets their restriction appeal granted.